src/components/SettingsPanel.vue

```diff
@@ -19,6 +19,22 @@ datapackStore.$subscribe(async (mutation, state) => {
 function randomizeSeed() {
     settingsStore.seed = random.nextLong()
 }
+
+function parseSeed(input: string): bigint {
+    if (/^\d+$/.test(input)) {
```

Comment:
This needs to accept a leading `-` and `+`.

Reply:
AAAAAAAHHHHH

Reply:
could you quickly check what happens with +/- in non-digit seeds? do i have to add it again ?

Reply:
I non-digit seeds it just treated as part of the string to get the hash of, so the current behavior is correct. :+1: